### PR TITLE
fix: Preventing Overflow 

### LINKED ISSUE #3164: Event Wizard: CSS body overflow in Tax Info Modal
**Describe the bug**

CSS Body Overflow in `Tax Info Modal`

**To Reproduce**

Steps to reproduce the behaviour:

1. Go to 'Create Event'
2. Click on 'Add Tax Information'
3. Scroll down to 'Business Name'

**Expected behaviour**

No Overflow

**Screenshots**

image

**Video**

https://drive.google.com/file/d/1QsxEsHpnKw4_QZGiO11Scus4azUT8mof/view?usp=sharing

**Desktop (please complete the following information):**

 - OS: MacOS
 - Browser: All

### PR DESCRIPTION
**Pull request by kushthedude**
Fixes #3164 

#### Changes proposed in this pull request:

- Introducing Scrolling to disable Overflow caused by `UI Modal`

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The acceptance, integration, unit tests and linter pass locally with my changes 
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

### CODE REVIEW

**Comment by kushthedude** [comment]
@shreyanshdwivedi @uds5501 Please Review !

File: app/components/modals/tax-info-modal.js [hunk 1]
```diff
@@ -49,56 +49,6 @@ export default ModalBase.extend(FormMixin, {
               prompt : this.l10n.t('Please give us your tax ID')
             }
           ]
-        },
-        taxInvoiceCompany: {
```
Comment (uds5501) [inline app/components/modals/tax-info-modal.js]: @kushthedude I don't think that the issue specified anywhere to remove the validations.

Comment (kushthedude) [reply app/components/modals/tax-info-modal.js]: @uds5501 The validations which I removed were of optional fields of which tick box was recently removed, I dont think this fields should be mandatory unless and until event organiser is a firm.
@shreyanshdwivedi @mrsaicharan1 Your views ?

**Comment by kushthedude** [comment]
@CosmicCoder96 @niranjan94 Please Review

**Comment by kushthedude** [comment]
@niranjan94 Removed all unwanted changes, Please Review

**Comment by kushthedude** [comment]
@CosmicCoder96 Review Please

**Pull request merged**